src/Screen/Fields/Relation.php

```diff
@@ -49,6 +50,7 @@ class Relation extends Field
         'relationSearchColumns' => null,
         'chunk'                 => 10,
         'allowEmpty'            => '',
+        'allowAdd'              => true,
```

Comment:
Is there any reason why this should be enabled by default?

Reply:
There is no reason... Fixed it, didn't check it right away)